### ISSUE TITLE
Game saves don't work

### ISSUE THREAD

**Issue by TheTrveAnthony** [raz0red/wii-mednafen#82]
Hi,

I have just starded using this emulator to play gba games instead of vbagx because the latter runs slowly.
Everything works fine, except game saves (real saves, not save-states), I have tried creating a "saves" directory (just like for vbagx) in sd/wiimednafen and also in sd/apps/wiimednafen, but nothing gets saved this way, so how is it supposed to work ? The info is not in the readme file, I also checked source code and couldn't find (I may have missed it)
Also, is it possible to load saves from vbagx (.sav files)

Thanks

**Comment by raz0red** [comment]
Hi-

I tested that functionality early on, but haven't tested it in some time. If you can let me know what game you are using, and how to trigger the save, I will give it a shot. If it is an issue, I can probably put together a quick build for you to fix it. I have a build that I have been working on for some time that has several other minor fixes, so this would be good to include as well if it is an issue.

Thanks, 
Chris

**Comment by TheTrveAnthony** [comment]
Thanks for your response,

I tried it with Pokemon brutal Version (ruby rom hack) and Castlevania Circle of the Moon. On both, I could load the save only after resetting the game (also pokemon said that the game file is corrupted), but not when I shut down the emulator and relaunched it, so I guess the save only goes to RAM and that's it. As I said before I created a "saves" dir as for vbagx and also a snes emulator but it doesn't work here.
I'll check if the same issue appears for other consoles and let you know.

**Comment by raz0red** [comment]
Ok, sounds good. I will give Castlevania a shot tonight and see what is happening.

Thanks,
Chris

**Comment by TheTrveAnthony** [comment]
Hi,

I have been trying to save other games:
On GB/GBC: Pokemon Blue, Pokemon Dark Energy (gold/silver rom Hack) and Metroid 2
on GBA: Sword of Mana, Breath of Fire and Metroid Zero Mission
on Mega Drive: Story of Thor

Saves from Mega Drive and GB/GBC games work fine. On GBA I had the same issue for every game I tested. 
Also save files are all stored in sd/wiimednafen/sav as ".sav" files, including for GBA games, so there must be something missing in GBA emulator.
Hope that helps

**Comment by raz0red** [comment]
Perfect, thanks a lot for digging into this. I should be able to look into tonight.

If not tonight, then over the next couple of days. Hopefully it is a straight forward fix, and I can get you an updated build.

Thanks again for all the work,
Chris

**Comment by TheTrveAnthony** [comment]
Hi,
Could you investigate on what is missing ?

**Comment by raz0red** [comment]
Yes, sorry for the delay. Work and real life has been taking up most of my free time recently.

I will take a look and let you know wheat I find in the next couple of days.

Thanks a lot,
Chris

**Comment by TheTrveAnthony** [comment]
That's fine, there's no hurry, I was simply wondering
